lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -191,25 +196,31 @@ def add_payment(post, payment, options)
       end
 
       def add_network_tokenization_card(post, payment, options)
-        post[:processingInformation][:paymentSolution] = PAYMENT_SOLUTION[payment.source]
         post[:processingInformation][:commerceIndicator] = 'internet' unless card_brand(payment) == 'jcb'
```

Comment:
commerceIndicator needs to be updated base on stored_credentials

Reply:
There's another PR that updates this value!